__tests__/app.test.js

```diff
@@ -200,4 +200,56 @@ describe('POST /api/articles/:article_id/comments', () => {
             expect(body.msg).toBe('Bad Request')
         })
     })   
-    })
\ No newline at end of file
+    })
+
+    describe('PATCH /api/articles/:article_id', () => {
+        test('200: responds with modified article', () => {
+            const newVote = 1
+            const patchObj = {inc_votes: newVote}
+            return request(app)
+            .patch('/api/articles/1')
+            .send(patchObj)
+            .expect(200)
+            .then(({body}) => {
+                expect(body).toEqual({title: "Living in the shadow of a great man",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "I find this existence challenging",
+                created_at: '2020-07-09T19:11:00.000Z',
+                votes: 101,
+                article_img_url:
+                  "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+              })
+            })
+        })
+        test('400: invalid article id', () => {
+            const newVote = 1
+            const patchObj = {inc_votes: newVote}
+            return request(app)
+            .patch('/api/articles/nonsense')
+            .send(patchObj)
+            .expect(400).then(({ body }) => {
+                expect(body.msg).toBe('Bad Request')
+            })
+        })
+        test('404: id not found', () => {
+            const newVote = 1
+            const patchObj = {inc_votes: newVote}
+            return request(app)
+            .patch('/api/articles/9999')
+            .send(patchObj)
+            .expect(404).then(({ body }) => {
+                expect(body.msg).toBe('Not Found')
+            })
+        })
+        test('400: incorrect body', () => {
+            const newVote = '1'
+            const patchObj = {inc_votes: newVote}
+            return request(app)
+            .patch('/api/articles/1')
+            .send(patchObj).then(({body}) => {
+                expect(body.msg).toBe('Bad Request')   
+            })
```

Comment:
You have forgotten to check the status code on this last test, but this is a good test suite overall well done!